bash/scripts/find_required_commits.sh

```diff
@@ -4,8 +4,8 @@ set -eo pipefail
 # find-required-commits parses a commit description for any required sibling
 # commits to pass along to downstream job(s)...
 find-required-commits() {
-  local git_commit="${1}"
-  local commit_description="${COMMIT_DESCRIPTION:-$(git log --format=%B -n 1 "${git_commit}")}"
+  git_commit="${1}"
```

Comment:
Was following a bad practice of declaring and assigning a `local` var in the same line -- and in fact this broke some logic.  No need for them to be `local` really, so removed.